specification/api-correlationcontext.md

```diff
@@ -1,24 +1,27 @@
-# DistributedContext API
+# Correlations API
 
-An `Entry` is used to label anything that is associated
-with a specific operation, such as an HTTP request.
+An `Entry` is used to label anything that is associated with a specific
+operation, such as an HTTP request.
 
-`DistributedContext` is an abstract data type that represents a collection of entries.
-Each key of `DistributedContext` is associated with exactly one value. `DistributedContext` is serializable,
-to facilitate propagating it not only inside the process but also across process boundaries.
-`DistributedContext` is used to annotate telemetry with the name:value pair `Entry`.
-Those values can be used to add dimension to the metric or additional context properties to logs and traces.
+`CorrelationContext` is an abstract data type that represents a collection of
+entries. Each key of `CorrelationContext` is associated with exactly one value.
+`CorrelationContext` is serializable, to facilitate propagating it not only
+inside the process but also across process boundaries. `CorrelationContext` is
+used to annotate telemetry with the name:value pair `Entry`. Those values can
+be used to add dimension to the metric or additional context properties to logs
+and traces.
 
-`DistributedContext` is a recommended name but languages can have more language-specific names like `dctx`.
+`CorrelationContext` is a recommended name but languages can have more
+language-specific names like `cctx`.
 
 ## Entry
```

Comment:
I think we should propose to remove `Entry` and `Metadata`. It's OK to describe "Name" and "Value" as an entry, but I wouldn't call it an `Entry`.

Reply:
Is your suggestion to remove Entry from the spec completely?

Reply:
Updated the PR to remove `Entry`